docs/man-openmpi/man3/MPI_Win_call_errhandler.3.rst

```diff
@@ -77,11 +77,7 @@ window.
 ERRORS
 ------
 
-Almost all MPI routines return an error value; C routines as the value
-of the function and Fortran routines in the last argument.
-
-See the MPI man page for a full list of MPI error codes.
-
+.. include:: ./ERRORS.rst
```

Comment:
Given the nature of this call, I think the original text is all that is needed.

Reply:
I put the above text back. However, this function does return an error status and the text above talks about error handling and overriding error handlers so I think the full text describing error handling is needed here. If I'm wrong I will remove it.

Reply:
Upon discussion, I think the new text is correct. The MPI Standard only talks about the success case. If the failure case (let's say you had a custom error handler that returns failure), then this would trigger the default error handler for that object - at least, that's my reading between the lines.